src/templates/plugins/prismic.js

```diff
@@ -86,7 +87,7 @@ export default async (context, inject) => {
       script.src = '<%= options.script %>'
       document.body.appendChild(script)
     })
-  }
+  }<% } %>
```

Comment:
I think this block could be moved right inside the `<% if (options.preview) { %>` block